### ISSUE TITLE
Check for attachments once discord.js 11.2.0 is released

### ISSUE THREAD

**Issue by anoadragon453** [Half-Shot/matrix-appservice-discord#47]
```
        };
        // Uncomment this and remove !hasAttachment above after https://github.com/hydrabolt/discord.js/pull/1449 pulled
        // if (hasAttachment) {
        //   hookOpts.file = opts.file;
        //   msg = await hook.send(embed.description, hookOpts);
        // } else {
        msg = await hook.send(embed.description, hookOpts);
// }
```

From: https://github.com/Half-Shot/matrix-appservice-discord/blob/develop/src/bot.ts#L206

https://github.com/hydrabolt/discord.js/pull/1449 is now merged, so maybe you want to uncomment it? :P

**Comment by Half-Shot** [comment]
This would require upgrading to 11.2.0, so I'll leave it until release 0.1.0 comes out.

**Comment by Half-Shot** [comment]
This has since been fixed in d123d5f086e581b1687c2e955567ee987b748638
